content/trademarks/haiku_icons.md

```diff
@@ -0,0 +1,17 @@
+# Haiku®'s Icons
```

Comment:
The whole point of this page being that the icons are, in fact, not trademarked, it seems a bit useless?

Reply:
@pulkomandy It seems this information is more suited to the Haiku Project website - maybe we can consider moving it there?